opennms-base-assembly/src/main/filtered/etc/opennms.service

```diff
@@ -9,6 +9,7 @@ User=opennms
 Environment="OPENNMS_HOME=${install.dir}"
 
 Type=forking
+RuntimeDirectory=opennms
```

Comment:
this is the magic juice... `systemd` will automatically create `/run/opennms` for us before the init script starts 👍